google/cloud/security/common/gcp_api/_base_client.py

```diff
@@ -41,8 +41,11 @@ def _attach_user_agent(request):
     Returns:
         HttpRequest: A modified googleapiclient request object.
     """
-    user_agent = request.headers['user-agent']
-    request.headers['user-agent'] = user_agent + ', %s/%s ' % (
+    user_agent = request.headers.get('user-agent', '')
+    if not user_agent or user_agent.contains(forseti_security.__package_name__):
```

Comment:
My thought was not muck with things if they didn't exist (too many possible reasons as to why that could not be set).

So only when it's set will we append to it.